bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -127,15 +138,39 @@ private void subscribeToActivePowerChanges() {
                     }
                 }
             }
-        }, 0, ENERGY_SUBSCRIPTION_PERIOD - 1, TimeUnit.MINUTES);
+        }, 0, POWER_SUBSCRIPTION_PERIOD - 1, TimeUnit.MINUTES);
+    }
+
+    private void subscribeToEnergyTotalizer() {
+        Where w = deviceWhere;
+        if (w == null) {
+            logger.warn("subscribeToEnergyTotalizer() WHERE=null. Skipping");
+            return;
+        }
+
+        energySchedule = scheduler.scheduleWithFixedDelay(() -> {
+            try {
+                send(EnergyManagement.requestCurrentDayTotalizer(w.value()));
+                send(EnergyManagement.requestCurrentMonthTotalizer(w.value()));
+
+            } catch (Exception e) {
+                logger.warn(
+                        "subscribeToEnergyTotalizer() Could not subscribe to totalizers scheduler for WHERE={}. Exception={}",
+                        w, e.getMessage());
+            }
+        }, 0, ENERGY_REFRESH_PERIOD, TimeUnit.MINUTES);
     }
 
     @Override
     public void dispose() {
-        if (notificationSchedule != null) {
-            ScheduledFuture<?> ns = notificationSchedule;
+        if (powerSchedule != null) {
+            ScheduledFuture<?> ns = powerSchedule;
             ns.cancel(false);
-            logger.debug("dispose() scheduler stopped.");
+            logger.debug("dispose() power scheduler stopped.");
+        } else if (energySchedule != null) {
```

Comment:
Maybe you should remove the "else"

Reply:
Yes you are right as powerSchedule and energySchedule are distinct reference variables so there is no need of _else_ keyword!

Reply:
Done